example/server.js

```diff
@@ -32,11 +32,17 @@ function init (port, pgConfig, callback) {
           return reply(listTags);
         });
       }
+    }, {
+      method: 'GET',
+      path: '/getAllActive',
+      handler: function (request, reply) {
+        request.getAllActive(function (error, allTags) { //eslint-disable-line
```

Comment:
Yeah.

Although would it make sense to attach these functions to a `.tags` object?

something like `request.pg.tags.getAllActive()`